From 866d91a0b6cf35dbb074b4a7233f0b6b9be24f79 Mon Sep 17 00:00:00 2001
From: Joni Poikelin <joni.poikelin@qt.io>
Date: Tue, 24 Nov 2020 18:11:14 +0200
Subject: [PATCH 16/19] Allow assimp to compile on non-gcc compilers

Gcc 4 version check can also be dropped since it is not supported
anymore.

Fixes: QTBUG-88821
Reviewed-by: Mike Krus <mike.krus@kdab.com>
(cherry picked from commit c6716fa7f044985389038802a296bae83951b119)

Change-Id: Ice522afc92823156e9545e8e75b16c2c0a614d6f
Reviewed-by: Mike Krus <mike.krus@kdab.com>
---
 src/plugins/sceneparsers/sceneparsers.pro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/plugins/sceneparsers/sceneparsers.pro b/src/plugins/sceneparsers/sceneparsers.pro
index ee48da565..a928764f5 100644
--- a/src/plugins/sceneparsers/sceneparsers.pro
+++ b/src/plugins/sceneparsers/sceneparsers.pro
@@ -2,7 +2,7 @@ TEMPLATE = subdirs
 # QNX is not supported, and Linux GCC 4.9 on ARM chokes on the assimp
 # sources (https://gcc.gnu.org/bugzilla/show_bug.cgi?id=66964).
 QT_FOR_CONFIG += 3dcore-private
-!ios:!tvos:!qcc:qtConfig(assimp):if(qtConfig(system-assimp)|android-clang|clang|win32-msvc):if(gcc:greaterThan(QT_GCC_MAJOR_VERSION, 4)) {
+!ios:!tvos:!qcc:qtConfig(assimp):if(qtConfig(system-assimp)|android-clang|clang|win32-msvc|gcc) {
     SUBDIRS += assimp
 }
 SUBDIRS += gltf
-- 
2.31.1

